### PR TITLE
Checkbox

### PR DESCRIPTION
**Pull request by PalomaAubeau**

### CODE REVIEW

File: app/ui/common/checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+export enum CheckboxColors {
```
Comment (MarshMallowFR) [inline app/ui/common/checkbox.tsx:1]: suggestion: you can reuse the enum ButtonColors in `app/ui/common/buttons.tsx`

File: app/ui/common/toggleInput.tsx [hunk 2]
```diff
@@ -1,4 +1,7 @@
-type ToggleInputColors = 'orange' | 'blue';
+export enum ToggleInputColors {
```
Comment (MarshMallowFR) [inline app/ui/common/toggleInput.tsx:1]: suggestion: same thing, we can reuse one enum for all the app.

**Review by MarshMallowFR — COMMENTED** [review]
I didn't see the fix when the all selected checkbox is checked and we unselect one line, the all selected checkbox has to be unchecked.

**Comment by PalomaAubeau** [comment]
> I didn't see the fix when the all selected checkbox is checked and we unselect one line, the all selected checkbox has to be unchecked.

> You can find it into ui/climbing/table.tsx :   useEffect(() => {
    const allSelectedCheck = Object.values(selectedStates).every(
      (isSelected) => isSelected,
    );
    setAllSelected(allSelectedCheck);
  }, [selectedStates]); 

The .every() method returns a boolean

File: app/lib/data.ts [hunk 3]
```diff
@@ -34,6 +34,7 @@ export async function fetchClimbPages() {
   noStore();
   try {
     const { rows } = await sql`SELECT id FROM sections WHERE name = 'climbing'`;
+    console.log(rows);
```
[the remark this answers is not in the document]
Comment (PalomaAubeau) [reply app/lib/data.ts:37]: Ok done

File: app/ui/common/checkbox.tsx (hunk 1 above)
Comment (PalomaAubeau) [reply app/ui/common/checkbox.tsx]: Yes good suggestion. I have create a type Color that is used each time it is needed.

File: app/ui/common/toggleInput.tsx (hunk 2 above)
Comment (PalomaAubeau) [reply app/ui/common/toggleInput.tsx]: Same answer :)

File: app/ui/climbing/select-members.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Checkbox } from '../common/checkbox';
+
+interface SelectMembersProps {
+  id: string;
+  isSelected: boolean;
+  setIsSelected: (isSelected: boolean) => void;
+}
+
+export default function SelectMembers({
```
[the remark this answers is not in the document]
Comment (PalomaAubeau) [reply app/ui/climbing/select-members.tsx:10]: Yes, right. Component SelecMembers has been deleted. Now that the "use-client" works for the Table, it's ok to pass functions into it and Checkbox component is enougth.

**Review by MarshMallowFR — APPROVED** [review]

**Pull request merged**